mmdet3d/core/bbox/coders/anchor_free_bbox_coder.py

```diff
@@ -0,0 +1,162 @@
+import numpy as np
+import torch
+
+from mmdet.core.bbox import BaseBBoxCoder
+from mmdet.core.bbox.builder import BBOX_CODERS
+
+
+@BBOX_CODERS.register_module()
```

Comment:
It seems a little bit similar to `PartialBinBasedBoxCoder`, clean duplicated code by inheritance or by adding options of `PartialBinBasedBoxCoder`